website/source/docs/guides/advanced-federation.html.md

```diff
@@ -7,12 +7,10 @@ description: |-
 ---
 
 # Multiple Datacenters
```

Comment:
```suggestion
# [Enterprise] Advanced Federation
```

Reply:
Same thoughts here as the basic federation.